src/github/pullRequestManager.ts

```diff
@@ -119,6 +124,47 @@ export class PullRequestManager implements IPullRequestManager {
 		}
 	}
 
+	private validateRemotesSetting() {
+		const remotesSetting = vscode.workspace.getConfiguration(SETTINGS_NAMESPACE).get<string[]>(REMOTES_SETTING);
+
+		if (remotesSetting) {
+			return remotesSetting.forEach(remote => {
+				if (!this._githubRepositories.some(repo => repo.remote.remoteName === remote)) {
+					vscode.window.showWarningMessage(`No remote with name '${remote}' found. Please update your 'githubPullRequests.remotes' setting.`);
+				}
+			});
+		}
+	}
+
+	private getDisplayedGitHubRemotes(): GitHubRepository[] {

```

Comment:
When I see the function name, I though the return results are `Remote`s but not `GitHubRepository`.